tests/CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
 project(tests)
 
-find_package(GTest REQUIRED)
-include(../3rdparty/FindGMock.cmake)
-find_package(Threads)
```

Comment:
Is the removal of `find_package(Threads)` a part of the fix, or is this a separate fix/accidental-removal?

Reply:
It's not really a fix, just an unneeded line given it's already specified in a higher level `CMakeLists.txt` (here: https://github.com/mit-dci/opencbdc-tx/blob/trunk/CMakeLists.txt#L25).

I can move this removal to a separate commit if you'd prefer that.

Reply:
> CC @AlexRamRam and @jonwiggins. I'll review this, but I don't have an apple machine available to test on. Would love to have a second set of eyes.

Other than #139, I am able to build and run the integration test on my Mac M1. Am I missing a part of the story?